src/test/java/org/broadinstitute/consent/http/service/UserServiceTest.java

```diff
@@ -906,6 +906,16 @@ void insertUserRoleAndInstitution_roleOnly() {
     verify(userRoleDAO).insertSingleUserRole(role.getRoleId(), testUser.getUserId());
   }
 
+  @Test
+  void insertServiceAccountUserRole() {
```

Comment:
Verify the service account can be assigned.

Reply:
Why test this role in particular? We don't have tests for the other roles, except `UserRoles.Researcher()` which is tested in `insertUserRoleAndInstitution_roleOnly()`.

If the intent is to test the database contents, as created by liquibase migrations (vs the behavior of `UserService`), then the test should loop over all the possible roles and test each.

Reply:
It's to test that we can create a service account without an institution because we shouldn't have to assign the institution for this path to function in our code.  This applies the `least privilege` principle to the service account user.

Reply:
Why do we need to test service account separately from researcher? More specifically, how can the code we're testing be broken in a way that researcher will work and service account won't?